luarules/gadgets/unit_death_animations.lua

```diff
@@ -15,37 +15,30 @@ if not gadgetHandler:IsSyncedCode() then
 end
 
 local hasDeathAnim = {
-  [UnitDefNames.corkarg.id] = true,
-  [UnitDefNames.corthud.id] = true,
-  [UnitDefNames.corstorm.id] = true,
-  [UnitDefNames.corsumo.id] = true,
-  [UnitDefNames.armraz.id] = true,
-  [UnitDefNames.armpw.id] = true,
-  [UnitDefNames.armck.id] = true,
-  [UnitDefNames.armrectr.id] = true,
-  [UnitDefNames.armrock.id] = true,
-  [UnitDefNames.armfast.id] = true,
-  [UnitDefNames.armzeus.id] = true,
-  [UnitDefNames.armfido.id] = true,
-  [UnitDefNames.armham.id] = true,
-  [UnitDefNames.corak.id] = true,
-  [UnitDefNames.corck.id] = true,
+	[UnitDefNames.corkarg.id] = true,
+	[UnitDefNames.corthud.id] = true,
+	[UnitDefNames.corstorm.id] = true,
+	[UnitDefNames.corsumo.id] = true,
+	[UnitDefNames.armraz.id] = true,
+	[UnitDefNames.armpw.id] = true,
+	[UnitDefNames.armck.id] = true,
+	[UnitDefNames.armrectr.id] = true,
+	[UnitDefNames.armrock.id] = true,
+	[UnitDefNames.armfast.id] = true,
+	[UnitDefNames.armzeus.id] = true,
+	[UnitDefNames.armfido.id] = true,
+	[UnitDefNames.armham.id] = true,
+	[UnitDefNames.corak.id] = true,
+	[UnitDefNames.corck.id] = true,
 }
 
-local dyingUnits = {}
-
 for udid, ud in pairs(UnitDefs) do --almost all raptors have dying anims
 	if string.find(ud.name, "raptor") or (ud.customParams.subfolder and ud.customParams.subfolder == "other/raptors") then
 		hasDeathAnim[udid] = true
 	end
 end
 
-local GiveOrderToUnit	= Spring.GiveOrderToUnit
-local SetUnitBlocking 	= Spring.SetUnitBlocking
-local UnitIconSetDraw   = Spring.UnitIconSetDraw
-local MoveCtrlEnable 	= Spring.MoveCtrl.Enable
-local MoveCtrlDisable 	= Spring.MoveCtrl.Disable
-local MoveCtrlSetVelocity = Spring.MoveCtrl.SetVelocity
```

Comment:
Aren't these speedups still relevant, as the engine functions are being called in `UnitDestroyed` and `RenderUnitDestroyed`, which can be called multiple times per frame?